src/util/Options.js

```diff
@@ -132,20 +140,37 @@ class Options extends null {
   }
 
   /**
-   * Create a cache factory using predefined limits.
-   * @param {Record<string, number>} [limits={}] Limits for structures.
+   * Create a cache factory using predefined settings to sweep or limit.
+   * @param {Record<string, SweptCollectionOptions|number>} [settings={}] Settings passed to the relevant constructor.
```

Comment:
```suggestion
   * @param {Object<string, SweptCollectionOptions|number>} [settings={}] Settings passed to the relevant constructor.
```
`Record` type doesn't exist in jsdoc